projects/ngrx-auto-entity/src/lib/service/error-handling.ts

```diff
@@ -0,0 +1,20 @@
+import { throwError } from 'rxjs';
+import { IEntityInfo } from '../actions/entity-info';
+
+export const notImplemented = (method: string, entityInfo: IEntityInfo): string =>
+  `Entity service method "${method}" has not been implemented. (Entity: ${entityInfo.modelName})`;
+export const notAFunction = (method: string, entityInfo: IEntityInfo): string =>
+  `Entity service method "${method}" is not a function. (Entity: ${entityInfo.modelName})`;
+
+export const logAndThrow = (method: string, err: any, entityInfo: IEntityInfo) => {
+  console.error(`[NGRX-AE] ! Service error: ${method}(). (Entity: ${entityInfo.modelName})`);
+  console.error(err);
+  return throwError({ info: entityInfo, err });
+};
+
+export const logServiceLocateFailure = (entityInfo: IEntityInfo, serviceName: string): void =>
+  console.error(
+    `[NGRX-AE] ! Error: Unable to locate service "${serviceName}" using model name of "${entityInfo.modelName}"`
```

Comment:
This message needs to be refactored at some point. Very early versions of auto-entity tried to find the service based on the model name, however that approach was abandoned in favor of the current provider mapping approach. This message should reflect that AN entity service, any entity service, could not be found for the specified model. (This fix can happen in a future branch.)